configuration-sample/ods-core.env.sample

```diff
@@ -369,3 +369,27 @@ ADDITIONAL_JIRA_PROJECT_TEMPLATE_PROPERTIES="# ---------------------------------
 
 # All params defined in the template https://github.com/opendevstack/ods-core/blob/master/ods-provisioning-app/ocp-config/dc.yml
 # can be overwritten by specifying the param in this file.
+
+########
+# Aqua #
+########
+# Is enabled of disabled aqua at cluster level
+AQUA_ENABLED=false
+
+# Is enabled for projects Aqua
+AQUA_ENABLED_FOR_PROJECT=true
```

Comment:
@jorge-romero @hrcornejo in our new implementation, a project is disabled selectively in the cluster-level ConfigMap. Is this configuration parameter still valid?

Reply:
That's true, it is not valid anymore, I have already remove it as the project level configuration done in ODS namespace will be done by the cluster admins.